MapboxNavigation/RouteMapViewController.swift

```diff
@@ -961,6 +964,7 @@ extension RouteMapViewController: StatusViewDelegate {
     }
 }
 
+//MARK: - RouteMapViewControllerDelegate
```

Comment:
This protocol already gets its own section in the jump bar menu. A documentation comment would be useful, however. 😉